spec/acceptance/st0305_support_script_and_upload_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper_acceptance'
+describe 'tasks' do
+  it 'run support script' do
+    run_shell('echo 192.69.65.61 customer-support.puppetlabs.net >> /etc/hosts')
+    result = run_bolt_task('support_tasks::st0305_support_script_and_upload', 'ticket' => 1234)
+    expect(result.stdout).to contain(%r{uploaded})
```

Comment:
Can we change this to check the exit code of the result?  The script exits with the return code of the sup script command, so we can use that to check for success. https://github.com/puppetlabs/support-tasks/blob/13c16da17c016de6947f8f1fed25a264e8f946f1/tasks/st0305_support_script_and_upload.rb#L37